WindowState/library/build.gradle

```diff
@@ -22,8 +22,6 @@ android {
     defaultConfig {
         minSdkVersion rootProject.ext.minSdkVersion
         targetSdkVersion rootProject.ext.targetSdkVersion
-        versionCode rootProject.ext.windowStateVersionCode
```

Comment:
Are you sure that we can remove these?

Reply:
I'm not 100%, they were removed when I ran the script to update the gradle version - I can look into this tomorrow